Package the python sdk (#8)
Theodo-UK/OmniLog: example/README.md

# OmniLog testing

## Dev

> To use the OmniLog package, you only need to duplicate the `omnilog` folder of `sdk-python/src` to the `example` folder.

Reviewer: ```suggestion
To use the OmniLog package, you only need to copy the `sdk-python/src` folder into the `example` folder.
```
